src/current/v25.1/deployment-operations-skills-taxonomy.md

```diff
@@ -0,0 +1,84 @@
+- [Handling unplanned node outages]({% link {{ page.version.version }}/recommended-production-settings.md %}#load-balancing)
+- [Adding nodes]({% link {{ page.version.version }}/cockroach-start.md %}#add-a-node-to-a-cluster)
+- [Removing nodes]({% link {{ page.version.version }}/node-shutdown.md %}?filters=decommission#remove-nodes)
+- [Add a region]({% link {{ page.version.version }}/alter-database.md %}#add-regions-to-a-database)
+- [Remove a region]({% link {{ page.version.version }}/alter-database.md %}#drop-region)
+- [Rolling upgrades]({% link {{ page.version.version }}/upgrade-cockroach-version.md %}#perform-a-patch-upgrade)
+- Downgrade a cluster from a [patch version]({% link {{ page.version.version }}/upgrade-cockroach-version.md %}#roll-back-a-patch-upgrade)
+- Downgrade a cluster from a [major version]({% link {{ page.version.version }}/upgrade-cockroach-version.md %}#roll-back-a-major-version-upgrade)
+- [Change a cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}#change-a-cluster-setting)
+- Cluster repaving involves the following individual skills, which are also used during [rolling upgrades]({% link {{ page.version.version }}/upgrade-cockroach-version.md %}#perform-a-patch-upgrade):
+    1. [Shut down a node gracefully]({% link {{ page.version.version }}/node-shutdown.md %})
+    1. Detach the [persistent volume]({% link {{ page.version.version }}/kubernetes-overview.md %}#kubernetes-terminology) (a.k.a. persistent disk) from the removed node's virtual machine (VM) (this step is optional but recommended)
+    1. Delete the removed node's VM
+    1. Start a new VM
+    1. Reattach the persistent disk to the new VM (necessary if you did step #2)
+    1. [Add a node to the cluster]({% link {{ page.version.version }}/cockroach-start.md %}#add-a-node-to-a-cluster) from the new VM
+
+## Troubleshooting
+
+- [SQL response time for specific queries]({% link {{ page.version.version }}/query-behavior-troubleshooting.md %}#query-issues)
+- [SQL throughput degradation across the board]({% link {{ page.version.version }}/query-behavior-troubleshooting.md %}#low-throughput)
+- [Cluster instability: Dead/suspect nodes]({% link {{ page.version.version }}/cluster-setup-troubleshooting.md %}#node-liveness-issues)
+- [Out of memory problems]({% link {{ page.version.version }}/cluster-setup-troubleshooting.md %}#out-of-memory-oom-crash)
+- [Imbalanced cluster load]({% link {{ page.version.version }}/architecture/replication-layer.md %}#load-based-replica-rebalancing)
+- [EOF errors]({% link {{ page.version.version }}/cluster-setup-troubleshooting.md %}#client-connection-issues)
+- [Changefeed is falling behind]({% link {{ page.version.version }}/advanced-changefeed-configuration.md %}#lagging-ranges)
+- [Get a "debug zip" file]({% link {{ page.version.version }}/cockroach-debug-zip.md %})
+- [Get a "tsdump" (timeseries dump) file]({% link {{ page.version.version }}/cockroach-debug-tsdump.md %})
```

Comment:
Similar to above, how about "Collect timestamped diagnostic logs" or similar? This one I'm less inclined to describe because that's a mess.

Reply:
i updated this one with a bit of description from the linked page so it now says

> Collect timeseries diagnostic data from a "tsdump" file 

Also went back and updated the debug zip one a little to use this type of phrasing